lib/cli.rb

```diff
@@ -106,8 +106,15 @@ def self.handle_error(e)
       Machinery::Ui.error("Error: " + e.message)
       exit 1
     else
-      Machinery::Ui.error "Machinery experienced an unexpected error. Please file a " \
-        "bug report at: https://github.com/SUSE/machinery/issues/new\n"
+      if LocalSystem.os.canonical_name.include? ("SUSE Linux Enterprise Server")
+        Machinery::Ui.error "Machinery experienced an unexpected error. Please file a " \
```

Comment:
Can you please change the wording to: `Machinery experienced an unexpected error. If this impacts your business please file a service request at https://www.suse.com/mysupport so that we can assist you on this issue. An active support contract is required.`